lib/cc/yaml/nodes/mapping.rb

```diff
@@ -75,10 +75,6 @@ def set_warnings(key)
 
       def []=(key, value)
         if mapped_key = mapped_key(key)
-          unless value.is_a? Node
-            node  = subnode_for(mapped_key)
-            value = node if Parser::Ruby.new(value).parse(node)
```

Comment:
Hmm... https://github.com/travis-ci/travis-yaml/blob/380b9db24a3cefa2b7fa3718ac49bd1db8761bfd/lib/travis/yaml/parser/ruby.rb

We also reference `Ruby::Parser` in `Sequence`, maybe I should just bring this file over and see if it all still works?